components/button/index.css

```diff
@@ -229,11 +229,16 @@ a.spectrum-Button {
   }
 
   &:disabled,
-  &.is-disabled {
+  &.is-disabled,
+  &[pending] {
```

Comment:
Interesting! I know that Spectrum Web Components is using a `pending` attribute there, but I don't think this is a valid HTML attribute for a `button`. Perhaps we could leave this selector, but also add an `.is-pending` selector, as well?

Reply:
Yeah that's correct, it's not an HTML attribute. I'll push that update!